examples/benchmark.py

```diff
@@ -182,46 +192,56 @@ def run(self, *args, **kwargs):
             warning("Could not load any results, nothing to plot. Exiting...")
             sys.exit(0)
 
-        gflopss = bench.lookup(params=parameters, measure="gflopss", event="loop_body")
-        oi = bench.lookup(params=parameters, measure="oi", event="loop_body")
-        time = bench.lookup(params=parameters, measure="timings", event="loop_body")
+        gflopss = bench.lookup(params=parameters, measure="gflopss", event="main")
+        oi = bench.lookup(params=parameters, measure="oi", event="main")
+        time = bench.lookup(params=parameters, measure="timings", event="main")
```

Comment:
It's not backward compatible for plotting, you're right. But I guess for comparison we're just interested in raw numbers, or at least something simpler to compare than roofline plots? Anyway, good point: I will add a switch